### DISCUSSION TITLE
Prerequisite for App Approval

### DISCUSSION THREAD

**Discussion by Jems5767** [etsy/open-api#1342] [Q&A]
I want to create a app for product listing management but when I requested it the request was rejected, i want to understand if requesting app access requires a live etsy store or are there any other prerquisite in order to request App access.

**Comment by minhichiba** [comment]
.

**Reply by Jems5767** [reply]
@minhichiba Do they ever respond ? As i see a lot of tickets opened.

**Comment by Jems5767** [comment]
i got response from the support team and they were able to help me solve my issue. Steps that you can follow.

1. Go to https://help.etsy.com/hc/en-gb/requests
2. Login and create a new request
3. Wait and you will hear from them in your mail.

Thank you, Etsy Support team.

@minhichiba you should try the above step.
